### PR TITLE
fix: temporarily remove transition before mount

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#38

### CODE REVIEW

File: src/Grid.ts [hunk 1]
```diff
@@ -299,12 +299,34 @@ abstract class Grid<Options extends GridOptions = GridOptions> extends Component
...
     this._im = new ImReady({
       prefix: this.options.attributePrefix,
     }).on("preReadyElement", (e) => {
       updated[e.index].updateState = UPDATE_STATE.WAIT_LOADING;
     }).on("preReady", () => {
+      mounted.forEach((item) => {
+        if (item.hasTransition) {
+          const element = item.element!;
+
+          element.style.transitionDuration = item.transitionDuration;
```
Comment (WoodNeck) [inline src/Grid.ts:329]: Looks like it's restoring the whole original CSS text in line 339, while it's only restoring transition duration here.
Is it okay to not restore the CSS text here?

**Review by WoodNeck — APPROVED** [review]
LGTM

File: src/Grid.ts [hunk 2]
```diff
@@ -299,12 +299,34 @@ abstract class Grid<Options extends GridOptions = GridOptions> extends Component
...
+
+        if (transitionDuration > 0) {
+          item.hasTransition = true;
+          item.transitionDuration = element.style.transitionDuration;
+          return true;
+        }
+      }
+      return false;
+    }).forEach((item) => {
+      item.element!.style.transitionDuration = "0s";
```
Comment (mixed) [inline src/Grid.ts:317]: I already checked that there is an element in the filter, do I need to put `!` again?

**Review by mixed — APPROVED** [review]
LGTM

File: src/Grid.ts (hunk 1 above)
Comment (daybrush) [reply src/Grid.ts:329]: 1. When an item is removed, there is no need to restore it.

2. Only the element was removed. The element is cached, and hasTransition and transitionDuration values are also maintained. This needs to be handled by InfiniteGrid.

File: src/Grid.ts (hunk 2 above)
Comment (daybrush) [reply src/Grid.ts:317]: Unfortunately, typescript doesn't even handle this.

**Pull request merged**